.changeset/modern-cooks-pay.md

```diff
@@ -0,0 +1,5 @@
+---
+'@monite/sdk-react': patch
+---
+
+feat(DEV-8990): improve InvoiceDetails screen. Support backend permissions for different buttons. Don't show some blocks if there is no data for it
```

Comment:
May I as you to split Changesets to separate MD files? The final result will look much better as three different features.

Reply:
What do you mean? Could you provide an example? I don't get it :)

Reply:
You mean call another `yarn changeset add` and based on that create another `.md` file with different message, is it correct?

Reply:
Sorry for the late answer. Yep, just for the beauty in the Changelog, it would be cool to see different bullets on different features.

Reply:
Splitted them into 2 different changesets and 2 different commits:
- 1644a535912136a32c9616f8b027bdc9e527702c
- https://github.com/team-monite/monite-sdk/pull/71/commits/cb9c44c1acbd8a75b52fa7d578eea055085ce29d